lib/const.rb

```diff
@@ -57,6 +57,27 @@
   HOME = CREW_PREFIX + Dir.home
 end
 
+CREW_IN_CONTAINER = File.exist?('/.dockerenv') || !ENV['CREW_IN_CONTAINER'].to_s.empty?
+
+CREW_CPU_VENDOR = CPUINFO['vendor_id'] || 'unknown'
+# vendor_id may not exist on non-x86 platforms.
+CREW_IS_AMD = ARCH == 'x86_64' ? CPUINFO['vendor_id'].include?('AuthenticAMD') : false
+CREW_IS_INTEL = ARCH == 'x86_64' || ARCH == 'i686' ? CPUINFO['vendor_id'].include?('GenuineIntel') : false
```

Comment:
Do we need `CREW_IS_ARM`?

Reply:
As previously mentioned, not at this time... Let's not add code which isn't needed right now, as that will slow things down.

Reply:
Sorry, I must have missed the previously mentioned.  Thought I forgot to submit the review.